src/walmart/WalmartProductList.js

```diff
@@ -17,8 +17,10 @@ class WalmartProductList {
     let products = [];
 
     productsJSON.forEach(function(product) {
-      if (product.hasOwnProperty('upc')) {
-        products.push(new WalmartProduct(product));
+      if (product.hasOwnProperty('upc') && product.hasOwnProperty('availableOnline')) {
```

Comment:
Could these two if statements be combined into one: `if (product.hasOwnProperty('upc') && product.availableOnline)` since `undefined` translates to false anyway?

Reply:
Fix added here 1f6e2b3